### DISCUSSION TITLE
写入学位授予单位 & 增加评阅人人数

### DISCUSSION THREAD

**Discussion by zhoujian9410** [nju-lug/NJUThesis#150] [Q&A] [answered]
博士学位材料包的国家图书馆封面的学位授予单位写有“南京大学”，而使用模板编译出来的却没有这一内容？如下图

不知道这一问题是否有比较容易的解决方法？没有的话我就直接用Adobe Acrobat进行手动修改

![一个小问题](https://user-images.githubusercontent.com/29673870/169643940-2d7707dd-5ad8-491d-a7c7-36467c8a778d.png)

**Comment by atxy-blip** [comment, marked as answer]
因为我参考的几份都没填，所以设计中就空着了。

这里南京大学四个字是固定不变的吧，也就是说只要填写授予日期就好？我明天加个设置项。

**Reply by zhoujian9410** [reply]
嗯嗯，南京大学字样是固定的。👍

**Reply by zhoujian9410** [reply]
抱歉，再次打扰一下，我刚试用了新的模板，但有两个小问题。
1. 实际上学位授予时间暂时未知，更希望编译之后是具体时间留空的“    年    月    日”字样。（学位授予单位固定为"南京大学"没有问题）
2. 评阅人实际上应该有5个，现在预留的是四个，我输入的最后一个教授无法显示

不知道这两个问题是否容易解决？

**Reply by atxy-blip** [reply]
在 `\begin{document}` 后 `\maketitle` 前加
```tex
\ExplSyntaxOn
\tl_set:Nn \g__nju_info_conferdate_tl {\qquad{}年\quad{}月\quad{}日} % 空日期串
\EditInstance{nju}{cover/nl/bottom}{ content = \__nju_cover_bottom_nl:n { 5 } } % 5个人
\ExplSyntaxOff
```
补丁是对当前版本生效的，不排除以后做一些更易用的修改。

**Reply by zhoujian9410** [reply]
太强了，多谢👍

**Reply by atxy-blip** [reply]
https://github.com/nju-lug/NJUThesis/commit/4a6c009660b9f33b1a934948690cfa3e37858943 提供了空白日期字样，按照 `confer-date={}` 这样的方式空着日期即可。
